sdk/core/azure-mgmt-core/sdk_packaging.toml

```diff
@@ -0,0 +1,9 @@
+[packaging]
+package_name = "azure-core"
```

Comment:
That's not the package name?

Reply:
This file isn't actually necessary, let me clean it for clarity

Reply:
Fixed